report_pre_thesis/src/sections/methodology.tex

```diff
@@ -206,4 +208,63 @@ \subsection{MOC}\label{sec:methodology_moc}
 \end{tabular*}
 \caption{Weighted Sum of Oxide Percentages. Elements marked with an asterisk (*) have been set to 50/50 as they are unspecified in \citet{cleggRecalibrationMarsScience2017}}
 \label{tab:weighted_sum_oxide}
-\end{table}
\ No newline at end of file
+\end{table}
+
+\subsection{Experiments}\label{sec:methodology_experiments}
+To evaluate the performance of each of the components in the pipeline, we focus our experiments on three main aspects:
+
+\begin{itemize}
+	\item \textbf{Outlier removal} to assess the impact of leaving outliers in the dataset or using a different outlier removal method.
+	\item \textbf{Hyperparameter tuning} to assess the impact of different hyperparameter configurations.
+	\item \textbf{Other models} to compare the performance of the PLS1-SM and ICA models to other models.
+\end{itemize}
+
+\noindent
+We recognize that these three aspects are commonly used to evaluate the performance and identify potential improvements of a machine learning pipeline.
+However, since the original authors did not conduct any experiments with alternative methods to demonstrate that their selected approach was superior, we feel that we have an incomplete picture of the potential performance of the pipeline.
+Additionally, a benefit of experimenting with alternative methods is that we can uncover which components contribute the most to the overall error and therefore would benefit the most from further research and development.
+
+% TODO: mention the reason we are doing experiments (to try and align our replication with the original pipeline as closely as possible)
+
+\subsubsection{Experiment: Outlier Removal}\label{sec:experiment_outlier_removal}
+The original PLS1-SM identified outliers manually by inspecting the leverage and spectral residuals plots.
+However, we have chosen to automate this based on the reasons described in Section~\ref{sec:methodology_outlier_removal}.
+It would therefore be interesting to see how the performance of the pipeline is affected by adjusting this process.
```

Comment:
You say interesting three times in three sentences. Consider more precise language?

Reply:
Rephrased.